tests/framework/e2e/etcdctl.go

```diff
@@ -176,3 +178,39 @@ func (ctl *EtcdctlV3) Compact(rev int64, o config.CompactOption) (*clientv3.Comp
 
 	return nil, SpawnWithExpect(args, fmt.Sprintf("compacted revision %v", rev))
 }
+
+func (ctl *EtcdctlV3) Grant(ttl int64) (*clientv3.LeaseGrantResponse, error) {
+	args := ctl.cmdArgs()
+	args = append(args, "lease", "grant", strconv.FormatInt(ttl, 10), "-w", "json")
+	cmd, err := SpawnCmd(args, nil)
+	if err != nil {
+		return nil, err
+	}
+	var resp clientv3.LeaseGrantResponse
+	line, err := cmd.Expect("ID")
+	if err != nil {
+		return nil, err
+	}
+	zap.NewExample().Error(line)
```

Comment:
This logging is not needed. Expect will write the output if you set environment variable (`EXPECT_DEBUG=true` if I remember correctly)

Reply:
Oh yup - this is leftover from debugging and apparently I missed it in my first cleanup pass 😅